### PR TITLE
Create Shahbaz.md

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Please tell want mistake I am doing.

**Comment by NishkarshRaj** [comment]
> 
> 
> Please tell want mistake I am doing.

Nothing. The Greetings GitHub Actions is failing because the template is broken currently.

File: intro/Shahbaz.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<p align="center">
+<img src="/intro/img/Shahbaz.jpg" height="400">
```
Comment (NishkarshRaj) [inline intro/Shahbaz.md:2]: This image must also be pushed to the repository.

**Review by NishkarshRaj — CHANGES_REQUESTED** [review]
Added comments. Fix and @ me 👍

**Pull request closed without merging**